Symmetrical Tribble/Assets/_Completed-Game/Scripts/GameManager.cs

```diff
@@ -70,16 +70,15 @@ public class GameManager : MonoBehaviour {
     void hideLevelImage() {
         levelImage.SetActive(false);
 
-
         doingSetup = false;
     }
 
     public void triggerBomb() {
-      GameObject bomb = GameObject.FindGameObjectWithTag("Bomb");
-      bomb.AddComponent<SeekBehavior>();
-      bomb.GetComponent<SeekBehavior>().target = GameObject.FindGameObjectWithTag("Player");
-      bomb.GetComponent<SeekBehavior>().speed = 3.0f;
-      bomb.GetComponent<SeekBehavior>().initialHeight = 25.0f;
+        GameObject bombInstance = Instantiate(bomb, new Vector3(0f,25f,0f), Quaternion.identity);
+        bombInstance.AddComponent<SeekBehavior>();
+        bombInstance.GetComponent<SeekBehavior>().target = GameObject.FindGameObjectWithTag("Player");
+        bombInstance.GetComponent<SeekBehavior>().speed = 3.0f;
+        bombInstance.GetComponent<SeekBehavior>().initialHeight = 25.0f;
 
       GetComponent<Collider>().gameObject.SetActive(false);
```

Comment:
This is the wrong collider and will throw an error. We should pass the collider object to the triggerBomb method, and then deactivate that.
```suggestion
      collider.gameObject.SetActive(false);
```